Gemfile

```diff
@@ -30,9 +31,8 @@ gem 'pundit', '~> 2.3'
 gem 'rack-cors', '~> 2.0'
 gem 'rswag-api', '~> 2.13.0'
 gem 'rswag-ui', '~> 2.13.0'
-gem 'sass-rails', '~> 6.0.0'
 gem 'sendgrid', '~> 1.2.4'
-gem 'sprockets', '~> 4.2.1'
+gem 'sprockets-rails', '~> 3.5', '>= 3.5.2'
```

Comment:
Why is this changing from `sprockets` to `sprokets-rails`?

Reply:
if we keep the `sprockets` gem and remove the `sass-rails`, we get this error `'require': cannot load such file -- sprockets/railtie`. Replacing it with the `sprockets-rails` one solves it. I'm not really sure why this happens

Reply:
Would be nice to understand why and what is the impact of this change. I assume we're good, but let's validate it doesn't have any negative impact

Reply:
just in case, `sprockets-rails` was already being installed as a dependency on `sass-rails`. https://github.com/rootstrap/rails_api_base/pull/807/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fL571-L573

Basically our previous setup wasn't so good, that's the whole issue here. `sprockets-rails` is required for our app that we are using the traditional asset pipeline.